### PR TITLE
customer.io docs, images and navigation

### PR DESCRIPTION
**Pull request by Adorism**
## Change description

New document for Customer.io integration with images and navigation 
### Development

- [X] Application changes have been tested appropriately

### Impact

- [ ] Code follows company security practices and guidelines
- [ ] Security impact of change has been considered
- [ ] Performance impact of change has been considered
- [ ] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [ ] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: pages/docs/integrations/grouparoo-customerio.mdx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  <tr>
+   <td>Destination Name
+   </td>
+   <td>Description
+   </td>
+  </tr>
+  <tr>
+   <td>
+
+<a href="customer.io-export-customers">Customer.io Export Customers</a>
```
Comment (teallarson) [inline pages/docs/integrations/grouparoo-customerio.mdx]: leads to a 404

File: pages/docs/integrations/grouparoo-customerio.mdx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+  </tablebody>
+</table>
+
+## Setup
+
+In order to sync data to Customer.io with Grouparoo, you will need a Customer.io account.
+
+During configuration of this integration using Grouparoo’s UI, you will need several to know how to connect to the API. Customer.io has two different types of API keys and you will need both for configuration. You can manage your account’s API keys at [here](https://fly.customer.io/settings/api_credentials) or, while signed in, navigate to **Account Settings > API credentials** where you can manage both Tracking API keys and App API keys.
+
+You can create a new site ID and Tracking API Key or copy/paste and existing pair. App API keys are only available to view upon creation.
```
Comment (teallarson) [inline pages/docs/integrations/grouparoo-customerio.mdx]:
```suggestion
You can create a new site ID and Tracking API Key or copy/paste an existing pair. App API keys are only available to view upon creation.
```

File: pages/docs/integrations/grouparoo-customerio.mdx [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+<table className="docTable">
+  <tablebody>
+    <tr>
+      <td>Option Name</td>
+      <td>Description</td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Side ID</strong> (required)
```
Comment (teallarson) [inline pages/docs/integrations/grouparoo-customerio.mdx]:
```suggestion
        <strong>Site ID</strong> (required)
```

File: pages/docs/integrations/grouparoo-customerio.mdx [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+      <td>Customer.io App API key</td>
+    </tr>
+  </tablebody>
+</table>
+
+## Customer.io Export Customers
+
+This Destination allows you to sync your Grouparoo Records to Customers in Customer.io.
+
+Companies use Customer.io to orchestrate data-driven communications through email, pugs notifications, and SMS messaging.
```
Comment (teallarson) [inline pages/docs/integrations/grouparoo-customerio.mdx]: push notifications?

File: pages/docs/integrations/grouparoo-customerio.mdx [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+
+When you [create a Destination](https://www.grouparoo.com/docs/config/destinations), you make choices about what data to sync.
+
+### Sync Modes
+
+- **Sync**: Add, update, and remove Customer.io Customers as needed.This option looks for new records, changes to existing records, and deletions.
```
Comment (teallarson) [inline pages/docs/integrations/grouparoo-customerio.mdx]:
```suggestion
- **Sync**: Add, update, and remove Customer.io Customers as needed. This option looks for new records, changes to existing records, and deletions.
```

**Review by teallarson — COMMENTED** [review]
Awesome!  1 broken link and a few nitpicks.

File: pages/docs/integrations/grouparoo-customerio.mdx (hunk 1 above)
Comment (Adorism) [reply pages/docs/integrations/grouparoo-customerio.mdx]: Oh, might be that dot issue. Will fix.

File: pages/docs/integrations/grouparoo-customerio.mdx (hunk 4 above)
Comment (Adorism) [reply pages/docs/integrations/grouparoo-customerio.mdx]: Oh my goodness, yes.

File: pages/docs/integrations/grouparoo-customerio.mdx [hunk 6]
```diff
@@ -0,0 +1,150 @@
+---
+title: "Customer.io"
+date: "2022-02-22"
```
Comment (edmundito) [inline pages/docs/integrations/grouparoo-customerio.mdx]: Update date before merge

File: pages/docs/integrations/grouparoo-customerio.mdx [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+  </tablebody>
+</table>
+
+## Customer.io Export Customers
+
+This Destination allows you to sync your Grouparoo Records to Customers in Customer.io.
+
+Companies use Customer.io to orchestrate data-driven communications through email, pugs notifications, and SMS messaging.
+
+When you [create a Destination](https://www.grouparoo.com/docs/config/destinations), you make choices about what data to sync.
```
Comment (edmundito) [inline pages/docs/integrations/grouparoo-customerio.mdx]: Documents on our website should just exclude the WWW and just have a path.

```suggestion
When you [create a Destination](/docs/config/destinations), you make choices about what data to sync.
```

File: pages/docs/integrations/grouparoo-customerio.mdx [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+  alt="Customer.io Export Customers Properties"
+  width={733}
+  height={378}
+/>
+
+<br />
+
+#### Required Properties
+
+You must map a Grouparoo Property to the **customer_id **Attribute in Customer.io. Choose the Grouparoo Property that uniquely identifies each individual customer’s record from the dropdown menu.
```
Comment (edmundito) [inline pages/docs/integrations/grouparoo-customerio.mdx]:
```suggestion
You must map a Grouparoo Property to the **customer_id** Attribute in Customer.io. Choose the Grouparoo Property that uniquely identifies each individual customer’s record from the dropdown menu.
```

File: pages/docs/integrations/grouparoo-customerio.mdx [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+
+In order to sync data to Customer.io with Grouparoo, you will need a Customer.io account.
+
+During configuration of this integration using Grouparoo’s UI, you will need several to know how to connect to the API. Customer.io has two different types of API keys and you will need both for configuration. You can manage your account’s API keys at [here](https://fly.customer.io/settings/api_credentials) or, while signed in, navigate to **Account Settings > API credentials** where you can manage both Tracking API keys and App API keys.
+
+You can create a new site ID and Tracking API Key or copy/paste an existing pair. App API keys are only available to view upon creation.
+
+## App
+
+To connect Customer.io to Grouparoo, you [create an App](https://www.grouparoo.com/docs/config/apps). This step will specify how Grouparoo can connect to Customer.io with your account information to prepare for setting up a new Destination in Grouparoo.
```
Comment (edmundito) [inline pages/docs/integrations/grouparoo-customerio.mdx]:
```suggestion
To connect Customer.io to Grouparoo, you [create an App](/docs/config/apps). This step will specify how Grouparoo can connect to Customer.io with your account information to prepare for setting up a new Destination in Grouparoo.
```

File: pages/docs/integrations/grouparoo-customerio.mdx [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+  alt="Customer.io Export Customers Optional Properties"
+  width={733}
+  height={162}
+/>
+
+<br />
+
+### Group Data
+
+Any groups you have made in Grouparoo would be available to use as a Customer.io Group Attribute. The Group Attributes you assign will help marketers engage with customers more effectively by applying meaningful segmenting to your company’s audience.
```
Comment (edmundito) [inline pages/docs/integrations/grouparoo-customerio.mdx]: "would be". Should it be "can be" or "will be" ?

**Review by edmundito — CHANGES_REQUESTED** [review]

File: pages/docs/integrations/grouparoo-customerio.mdx (hunk 10 above)
Comment (Adorism) [reply pages/docs/integrations/grouparoo-customerio.mdx]: "Will be" is probably more clear. I'll update that.

File: pages/docs/integrations/grouparoo-customerio.mdx (hunk 6 above)
Comment (Adorism) [reply pages/docs/integrations/grouparoo-customerio.mdx]: updated!

**Review by edmundito — APPROVED** [review]

**Pull request merged**